src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/util/ValidationUtil.groovy

```diff
@@ -75,6 +76,30 @@ class ValidationUtil {
         return returnValue
     }
 
+    /**
+     * Port of FormValidation.validateIntegerInRange for Double.
+     *
+     * Make sure that the given string is an double in the range specified by the lower and upper bounds (both inclusive)
+     *
+     * @param value the value to check
+     * @param lower the lower bound (inclusive)
+     * @param upper the upper bound (inclusive)
+     */
+    static FormValidation validateDoubleInRange(String value, double lower, double upper) {
```

Comment:
* `min` and `max` instead of `lower` and `upper` as args 
* but the description in context of bound its fine

Reply:
Wanted to keep is as close to the original funktion I adapted